l10n_br_account_payment_order/models/account_move.py

```diff
@@ -56,50 +56,59 @@ def get_invoice_fiscal_number(self):
 
     def action_post(self):
         result = super().action_post()
+        self.load_cnab_info()
```

Comment:
Como foi feito um método especifico para o caso do CNAB acredito que a validação para chama-lo ou não deveria ser feita antes e não dentro dele, incluindo antes dessa linha um:

if self.payment_mode_id and self.payment_mode_id.payment_method_code in BR_CODES_PAYMENT_ORDER:

Reply:
Eu pensei em separar essa lógica para daqui a pouco, caso necessite chamar a logica de criação do boleto, não seja necessário repetir código, se eu por essa validação fora, todo lugar que eu chamar o load_cnab_info() será preciso incluir essa validação, como não há possibilidade de criar o CNAB caso essa validação seja false, optei por deixar já dentro do método. Penso no futuro criar um botão "Regerar Boleto" para os caso que o Payment Mode for modificado.